docs/unique/debugging-broker-connections.md

```
@@ -0,0 +1,1029 @@
+---
+It's been a bit of a head scratcher today... ^ref-73d3dbf6-1-0
+
+There is a process that seems to be connecting repeatedly to the broker, but it isn't crashing. ^ref-73d3dbf6-3-0
+
+The broker logs don't provide enough information for me to identify what process is doing that. ^ref-73d3dbf6-5-0
+
+The embeddings seem to be empty often? ^ref-73d3dbf6-7-0
+
+I need an easier way to figure out who is doing what. I only get session IDs right now, and they are not helpful. ^ref-73d3dbf6-9-0
+
+we fixed the shared typescript packages compiling wierdly. It should be fixed for good now, as long as no one tries to use a relative path to import a typescript file from outside of it's module. ^ref-73d3dbf6-11-0
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 120; Actual: 214

(MD013, line-length)